src/Raven.Client/Documents/Operations/Backups/BackupConfiguration.cs

```diff
@@ -0,0 +1,127 @@
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+using Sparrow.Json.Parsing;
+
+namespace Raven.Client.Documents.Operations.Backups
+{
+    public class BackupConfiguration : IDynamicJson
+    {
+        public string Name { get; set; }
```

Comment:
what is this used for in one time backup?

Reply:
we use the base class `BackupConfiguration` in BackupTask so it should contain Name

Reply:
I do not see a reason for that. If you need a name on the server-side then put a well known human readable name there.